try.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+published=none
+for i in {1..3}; do
+  online_package=$(nuget list Piiano.Vault)
+  echo "Published: ${online_package}, Expecting: "
+  published_ver=$(echo $online_package |  awk '{ print $2 }')
+  if [ ${published_ver} != "1.0.6" ] ; then
```

Comment:
Can you make the version as param to the script?

Reply:
This file got in by mistake. I will remove it